fe/fe-core/src/main/java/org/apache/doris/nereids/stats/StatsCalculator.java

```diff
@@ -495,8 +495,12 @@ private Statistics computeOlapScan(OlapScan olapScan) {
             // mv is selected, return its estimated stats
             Optional<Statistics> optStats = cascadesContext.getStatementContext()
                     .getStatistics(((Relation) olapScan).getRelationId());
+            LOG.info("computeOlapScan optStats isPresent {}, tableRowCount is {}",
```

Comment:
better change to LOG.debug? to avoid too many log printing?

Reply:
This log will only be printed when the materialized view is hit, so it shouldn't occur frequently. Additionally, this is for troubleshooting an occasional issue in the pipeline. If set to debug, it may cause us to miss relevant information.